ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -62,7 +64,7 @@ QObject {
             FastExpressionRole {
                 name: "colorizedChainPrefixes"
                 function getChainShortNames(chainIds) {
-                    const chainShortNames = root.getNetworkShortNames(chainIds)
+                    const chainShortNames = WalletUtils.getNetworkShortNames(chainIds, root.filteredFlatNetworksModel)
```

Comment:
Looks like the same thing repeated here and `WalletAccountsAdaptor`. Maybe accounts model could be decorated in the first step on the beginning and then used in that form by other adaptors/views.

Reply:
I already created a task for looking at this repetition here: https://github.com/status-im/status-desktop/issues/15473